Modules/xASL_module_ASL.m

```diff
@@ -577,6 +577,10 @@
 
 
 %% 1. Load ASL parameters (inheritance principle)
+
+% First ensure that we don't use quantification parameters from other sequences
+x.Q = struct;
```

Comment:
From previously. We have a rather complicated system of saving things to x and then loading them. Which includes also structural data that are either added here after the structural module or reloaded. 

Completely deleting all that's in x.Q is likely going to have important negative consequences!

Reply:
I checked this, x.Q is only used for ASL quantification parameters, that we always reload after this line from ASL4d.json sidecars etc. And we had this already, just not at this earlier point. So we don't change things except from ensuring that settings from previous module runs are not propagated to new module runs (each module run should be independent of other module runs).